include/awkward/kernel.h

```diff
@@ -1,14 +1,1555 @@
 // BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/master/LICENSE
 
-#ifndef AWKWARD_KERNEL_H
-#define AWKWARD_KERNEL_H
+#ifndef AWKWARD_KERNEL_H_
+#define AWKWARD_KERNEL_H_
 
 #include "awkward/common.h"
 
 namespace kernel {
-    template <typename T>
-    ERROR new_identities(T *toptr,
-                         int64_t length);
+  class Identities;
+  template <typename T>
+  class IndexOf;
```

Comment:
Why are the Identities and IndexOf classes forwarded here? They're not used in this file. (Kennel function definitions don't reference classes.)

Reply:
Oh, that's a residue from my copying stint from `util::`, I'll remove them in the next commit.